govcd/nsxt_edgegateway_bgp_configuration.go

```diff
@@ -0,0 +1,78 @@
+/*
+ * Copyright 2022 VMware, Inc.  All rights reserved.  Licensed under the Apache v2 License.
+ */
+
+package govcd
+
+import (
+	"fmt"
+
+	"github.com/vmware/go-vcloud-director/v2/types/v56"
+)
+
+func (egw *NsxtEdgeGateway) GetBgpConfiguration() (*types.EdgeBgpConfig, error) {
```

Comment:
Shouldn't we add documentation for the public functions?

Reply:
Totally - this is my oversight. Fixed.